### DISCUSSION TITLE
Improved sql import and export for entity uml diagrams.

### DISCUSSION THREAD

**Discussion by lastlink** [jgraph/drawio#3095] [Ideas]
In reference to issue #3083. Also similar to #2806 

Would like to be able to **import** mysql, sqlserver, sqlite, and postgres into drawio as well as **export** sql.
I've already done prs as extensions that make this possible. Idk if they need to be rewritten. Would like to get these added into a release soon mainly b/c would like an alternative over lucidcharts for database diagraming.

@davidjgraph

**Comment by davidjgraph** [comment]
Did you read the bottom of the README?

**Reply by lastlink** [reply]
Yes I read it, closed contributions, that's why I mentioned that you might be **rewriting** to meet your standards if you decide to add it.

**Reply by davidjgraph** [reply]
OK, we'll leave it open in case a team member wants to look at it.

**Comment by lastlink** [comment]
@davidjgraph any updates from the team on when this sql plugin may be available?

**Reply by davidjgraph** [reply]
I don't know, if someone wants to take on the task I assume they will make it known in the thread.

**Comment by lastlink** [comment]
Update pr #4122
